documentation/types/type-messages.md

```diff
@@ -22,21 +22,21 @@ The body of a Type message consists of an array of objects. The following keywor
 | `extrapolation` | Optional data mode used to provide consistency when reading values. Supported values include `all`, `none`, `forward`, and `backward`. |
 | `properties` | Key-value pairs defining the properties of a static or dynamic Type. Required unless the Type defines enum. |
 
-The `id` cannot begin with the character sequence __. This has been reserved for predefined Types. Currently the only supported predefined Type 
-is [__Link](xref:linkType). The `id` property is referenced when creating instances of Types in Container and Data messages, or when 
+The `id` cannot begin with the character sequence __. This has been reserved for predefined Types. Currently the only supported predefined Type
+is [__Link](xref:linkType). The `id` property is referenced when creating instances of Types in Container and Data messages, or when
 creating other Types that include this Type as a referenced Type.
 
-The `version` is used to supply information about the Type definition and is stored as meta data. 
+The `version` is used to supply information about the Type definition and is stored as meta data.
 
 A `static` classification represents metadata describing a device being observed and should be used to capture data that is descriptive and
```

Comment:
31 and 32 should be merged into one line too

Reply:
Merged